fiftyone/plugins/context.py

```
@@ -110,6 +110,66 @@ def _ensure_parent_modules(module_name, plugin_dir):
+        return
+
+    for dirpath, dirnames, _ in fou.walk_safe(root, follow_symlinks=True):
+        if "__pycache__" in dirnames:
+            shutil.rmtree(
+                os.path.join(dirpath, "__pycache__"),
+                ignore_errors=True,
+            )
+            dirnames.remove("__pycache__")
+
+
+def _invalidate_plugin_python_caches(plugin_dir):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_invalidate_plugin_python_caches`

Add return type annotation: `None`

(ANN202)